test/Microsoft.ApplicationInsights.AspNet.Tests/Extensions/ApplicationInsightsExtensionsTests.cs

```diff
@@ -99,6 +82,130 @@ public static void RegistersTelemetryConfigurationFactoryMethodThatReadsInstrume
             }
 
             [Fact]
+            public static void RegistersTelemetryConfigurationFactoryMethodThatReadsDeveloperModeFromConfiguration()
+            {
+                var services = ApplicationInsightsExtensionsTests.GetServiceCollectionWithContextAccessor();
+                var config = new Configuration().AddJsonFile("content\\config.json");
```

Comment:
We are sharing the config.json as a test fixture in multiple tests now. Can we change this to use the [InMemoryConfigurationSource](https://github.com/aspnet/Configuration/blob/dev/src/Microsoft.Framework.ConfigurationModel/Sources/MemoryConfigurationSource.cs) and setup configuration separately for each of these tests?